### ISSUE TITLE
Missing Quest in my questbook called "Reforging The Sword (Alliance)", But the item is in my bag

### ISSUE THREAD

**Issue by frahheh** [dalaranwow/dalaran-wow#5518]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I bought a battered hilt item from the horde actionhouse when my character was still leveling because i found it for a cheap price, once i reached 80, i started to complete the questchain untill the point when you're supposed to enter PoS, but i did not like the Horde side and also i was not attuned yet,, so i moved on to the Alliance side. So i got my attunement on the Alliance side but once i entered pos, to my supprise the quest was missing but the item is in my bag, you're supposed to enter Pos and you must gather bars of infused saronite(5) and capture the forgemaster's hammer, then bring them to the forgemaster's anvil and re-forge the sword. I have the item in my bag "Remnants of Quel'Delar", but the quest is missing, so i cannot pick up any saronite ores to continue. 

"Quest:Reforging The Sword (Alliance)"--(missing)

**Expected behaviour**: Tell us what should happen instead
I should have the quest in my Questbook called "Quest:Reforging The Sword (Alliance)" So 5 saronite bars should be glowing inside pos so i can pick it up and continue the questline.
**Steps to reproduce the problem**:

1. Contact my ingame, i had a ticket up for soon two weeks. Nick: Frah
2. Please refresh/reset my quest? Something got bugged out untill i got the attunement ready, and the item is still in my bag, but the quest is nowhere to be seen...
3. 

**Include proofs for this behaviour**
Jewelcrafting, Engineering.
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Check the correct NPC and re-take the quest so that you can proceed through the quest chain.
Closing
